app/validators/bookmark_controller_validator.rb

```diff
@@ -0,0 +1,7 @@
+class BookmarkControllerValidator < ActiveModel::EachValidator
+  @@active_record_tables ||= ActiveRecord::Base.connection.tables.map(&:to_s)
```

Comment:
Using `||=` here, I don't think it'll get run twice.  Did you mean to put this in a class method (you could call that from the validator below then)?  Or just set it once (`=`).